src/components/ActionDisplay/ActionDisplay.vue

```diff
@@ -1,10 +1,14 @@
 <template>
-  <span class="col2-item bottom overflow_ellipsis">
-    <span v-if="transaction.type === transactionType.TRANSFER">-></span>
-    <span>{{ descriptor }}</span>
-
-    <span v-if="needsCosignature" class="click-to-cosign">({{ $t('Click_to_cosign') }})</span>
-  </span>
+  <div>
+    <span class="col2-item overflow_ellipsis"><AddressDisplay :address="transaction.signer.address" /></span>
+    <span class="col2-item bottom overflow_ellipsis">
+      <span v-if="transaction.type === transactionType.TRANSFER"> ->
+        <AddressDisplay :address="transaction && transaction.recipientAddress" />
```

Comment:
I didn't understand the type of prop `address`  is ` Address | NamespaceId`, but   the value of `transaction && transaction.recipientAddress`  is `Boolean`

Reply:
ACK